src/domain/constant/about/aboutTextElements.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import LiveSuperscript from '@/components/About/LiveSuperscript';
+import StyledLink from '@/components/About/StyledLink';
+import { SearchOperations } from '@/operations/Search/SearchOperations';
+
+export const aboutTextElements = SearchOperations.makeTextElementsSearchable([
```

Comment:
do same as other exported default elements to stay consistent and its much easier for non coders
```typescript
const aboutTextElements = ...
...
...
export default SearchOperations.makeAccordionItemsSearchable(aboutTextElements);